lib/std/special/init-exe/src/main.zig

```diff
@@ -1,7 +1,8 @@
 const std = @import("std");
 
-pub fn main() anyerror!void {
-    std.log.info("All your codebase are belong to us.", .{});
+pub fn main() !void {
+    const stdout = std.io.getStdOut().writer();
+    try stdout.print("All your codebase are belong to us.", .{});
```

Comment:
missing `\n` but otherwise looks good once the tests pass

Reply:
Yeah, that probably should be there, shouldn't it. I'll add it when I fix the tests.

Reply:
Cheers. All fixed.